doc/connectivity/bluetooth/api/mesh/rpr_cli.rst

```diff
@@ -0,0 +1,25 @@
+.. _bluetooth_mesh_models_rpr_cli:
+
+Remote Provisioning Client
+##########################
+
+The Remote Provisioning Client model is a foundation model defined by the Bluetooth
+mesh specification. It is enabled with the
+:kconfig:option:`CONFIG_BT_MESH_RPR_CLI` option.
+
+The Remote Provisioning Client model is introduced in the Bluetooth Mesh Protocol
+Specification version 1.1, and provides functionality to remotely provision devices into a mesh
+network by interacting with mesh nodes that supports the :ref:`bluetooth_mesh_models_rpr_srv` model.
```

Comment:
It adds more features like NPPI(or how it is called in the last spec version?) interface to change composition data, address and device key. Probably it is worth to add a bit more about these abilities since they are quite important. 
Also, it adds extended scanning (that includes active scanning) but I'm not sure if it is worth to go so deep.